core/tests/puppeteer-acceptance-tests/spec/practice-question-reviewer-tests/review-and-accept-question.spec.js

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Acceptance Test for question admins to add reviewing
+ * question rights to users.
+ */
+
+const userFactory = require(
+  '../../puppeteer-testing-utilities/user-factory.js');
+const testConstants = require(
+  '../../puppeteer-testing-utilities/test-constants.js');
+
+const DEFAULT_SPEC_TIMEOUT = testConstants.DEFAULT_SPEC_TIMEOUT;
+
+describe('Question Admin', function() {
+  let questionAdmin = null;
+
+  beforeAll(async function() {
+    questionAdmin =
+      await userFactory.createNewQuestionAdmin('questionAdm');
+  }, DEFAULT_SPEC_TIMEOUT);
+
+  it('should be able to provide rights to review and submit questions to user.',
+    async function() {
+      let reviewer = await userFactory.createNewGuestUser(
+        'reviewer', 'review@example.com');
+      await userFactory.closeBrowserForUser(reviewer);
+      await questionAdmin.navigateToContributorDashboardAdminPage();
```

Comment:
Let's modify this whole test (and the associated utilities) so that the top-level reads as follows:

```
      await questionAdmin.navigateToContributorDashboardAdminPage();
      await questionAdmin.verifyQuestionReviewersExcludeUser('reviewer');   // this checks the list of question reviewers

      await questionAdmin.addReviewQuestionRights('reviewer');

      await questionAdmin.verifyUserCanReviewQuestions('reviewer');   // this checks the user's roles
      await questionAdmin.verifyQuestionReviewersIncludeUser('reviewer');  // this checks the list of question reviewers
```

Reply:
Done